packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -87,6 +100,24 @@ export function Menu({
     </div>
   );
 
+  function setRef(
```

Comment:
I'm not following why we need this method?

Reply:
We were set the refs in a DRY way, but now that we only need the ref for the menu, we don't need this anymore! Killed this method ⚔️